packages/utils/src/RIP7712NonceManagerUtils.ts

```diff
@@ -0,0 +1,16 @@
+import { JsonRpcProvider } from '@ethersproject/providers'
+
+import { bytecode as nonceManagerByteCode } from '../artifacts/@account-abstraction/rip7560/contracts/predeploys/NonceManager.sol/NonceManager.json'
+import { DeterministicDeployer } from './DeterministicDeployer'
+import { NonceManager__factory } from './types/factories/@account-abstraction/rip7560/contracts/predeploys'
+
+export const entryPointSalt = '0x90d8084deab30c2a37c45e8d47f49f2f7965183cb6990a98943ef94940681de3'
+
+export async function deployNonceManager (provider: JsonRpcProvider, signer = provider.getSigner()): Promise<any> {
```

Comment:
you don't need any salt for nonceManager (or if you like, you can find a better salt.).

Reply:
The code or address of the NonceManager is probably not final yet. I think it is okay to find a suitable nonce sometime later. I don't want to remove the salt altogether for now in case we actually decide to mine for the better salt later.